backend/src/API/memberAPI.ts

```diff
@@ -169,3 +170,81 @@ export const generateMemberArchive = async (
   }
   return MembersDao.generateArchive(membershipChanges, semesters);
 };
+
+/**
+ * Checks if a member meets a category.
+ * @param otherMemberEmail - the email of the member we are checking.
+ * @param submitter - the member that submitted the coffee chat.
+ * @param encodedCategory - the category we are checking with (encoded with base64).
+ * @returns true if a member meets a category, false if not, undefined if not enough data to know.
+ */
+export const checkMemberMeetsCategory = async (
```

Comment:
Sorry, what I meant was, can you move this function into `coffeeChatAPI`, delete `getMemberProperties` from `coffeeChatAPI` (since we don't want to expose the functionality of actually showing a user's properties), and then just call the `CoffeeChatDao.getMemberProperties` function directly from `checkMemberMeetsCategory`?